check_source.py

```diff
@@ -356,6 +364,15 @@ def source_has_correct_maintainers(self, source_project):
 
         return self.required_maintainer in maintainers
 
+    def select_add_role(self, request):
```

Comment:
This filter has nothing to do with the request being an add_role one, but with the person assigned to the action. So I would change the name to something like `select_required_maintainer`.

Reply:
Please, write a sentence to describe what the function returns